src/test/resources/template/ParameterReuseRecipe$1_before.java

```diff
@@ -16,7 +16,19 @@
 package foo;
 import org.openrewrite.java.*;
 
+/**
+ * OpenRewrite `before` template created for  `foo.ParameterReuseRecipe$1`.
```

Comment:
NB: Since this is JavaDoc it'd be more accurate to use `{@code ...}` rather than backticks. (Same applies to #51.)

Reply:
Thanks! Applied here already. Now on to #51.